apps/experiments/export.py

```diff
@@ -10,9 +10,28 @@ def _parse_tags(tags: list[Tag]) -> str:
     return ", ".join([t.name for t in tags])
 
 
+def _parse_comments(user_comments: list[UserComment]) -> str:
+    """Parses `user_comments` into a single string that looks like this:
```

Comment:
I think `parse` is the wrong term here. It should be `format`: `_format_comments`

Reply:
Ah, I used it wrong all the time, thanks! :sweat_smile: